src/hooks/useClipboard.tsx

```diff
@@ -0,0 +1,11 @@
+const useClipboard = () => {
+  const setClipboard = (text: string) => {
```

Comment:
copyToClipboard is a more common name.
- setClipboard requires more thought to understand what the function does
```suggestion
  const copyToClipboard = (text: string) => {
```